utils/travis/mp_test_executor.sh

```diff
@@ -12,10 +15,49 @@ hostpid=$!
 ./wesnoth --plugin=join.lua --server=localhost:12345 --username=join --mp-test --noaddons --nogui &
 joinpid=$!
 
-wait $hostpid
+START_TIME=$SECONDS
+HOST_RUNNING=0
+JOIN_RUNNING=0
+while true; do
+    # Timeout
+    EXEC_TIME=$(($SECONDS - $START_TIME))
+    if [ $EXEC_TIME -gt $TIMEOUT_TIME ]; then
+        kill $hostpid 2>/dev/null
+        kill $joinpid 2>/dev/null
```

Comment:
For consistency, I'd prefer if you added a `break` statement here as well.

Reply:
Looks like I missed that one. Added.